lib/async_hooks.js

```diff
@@ -45,9 +45,9 @@ class AsyncHook {
     if (before !== undefined && typeof before !== 'function')
       throw new errors.TypeError('ERR_ASYNC_CALLBACK', 'before');
```

Comment:
How about passing `'hook.before'` as the option name? Usually a bare identifier means a function parameter.

Reply:
Sounds good, I'll take a look. Thanks